### ISSUE TITLE
create poster sessions (one file per poster)

### ISSUE THREAD

**Issue by kenirwin** [alaoweb/2021.alaoweb.org#78]
* in the _posts folder
* can we do more to leverage the _data/speakers.yml data rather than relying on the "presenters" data block in the individual poster files? I'm going to see what I can do with that.

**Comment by kenirwin** [comment]
contents for files:
```
---
layout: poster
title: Cyber House Rules: how to host an winning virtual game night
description: Our institution has hosted monthly board game nights for our campus community for the last several years. However, when the COVID-19 pandemic hit, we were forced to cancel our in-person events and rethink how we would deliver game nights going forward. While pivoting to a virtual event seemed like the obvious solution, we encountered numerous setbacks in our quest to make an online game night a reality. In spring 2020, we experimented with online gaming platforms in order to ascertain the feasibility of holding virtual library game nights. This included a pilot event with library staff. Throughout these experiments we ran into many challenges, including: fickle internet speeds and computing power; a steep learning curve for the online platforms we were interested in using; limitations of the software available to us at the time; and technological comfort levels of the committee members. These impediments were extreme enough to delay our second attempt until spring of 2021, after a year-long hiatus. We will discuss how we worked to overcome these challenges for our second attempt, and will give attendees the opportunity to reflect on how they can host similar outreach events in their own institutions.
date: 2021-07-01 08:00:00
audience: PROMIG
keywords: gaming, service transitions, student engagement, virtual event planning
presenter-names: Abigail Morgan,Ginny Boehme,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Visualizing interlibrary loan data to reveal user needs
description: "This poster will explore the results of a project to analyze and visualize data showing interlibrary loan requests for journals that an Ohio academic library made over a 4 year span. The presenter visualized the data in Tableau and set out to answer questions such as:\n\n* Can we use this data to expose gaps in the diversity of our journal collection?  \n* Are journal titles showing up in this data that were cancelled in past content cuts?  \n* Can this data help to inform future journal subscription acquisitions? \n\nThe poster will cover the tools and techniques used to analyze and visualize the data. It will also show patterns revealed by the data that provide insight into user needs that aren’t being met by the current collection."
date: 2021-07-01 08:00:00
audience: AIG, CMIG, TEDSIG
keywords: Interlibrary loan; Data visualization; Data analysis; Collection development
presenter-names: Shelby Stuart,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Teaching to the Same Audience: Collaboration between Instruction and Special Collections Teams
description: One core value of academic library work is collaboration; done effectively, collaboration provides superior service to students, faculty and researchers.  Recently, an archivist and a librarian conducted a grant-funded research project that aimed to identify the levels of collaboration that currently exist for library instruction between special collections and instruction teams. Increasingly archivists and special collections librarians are incorporating primary source instruction into their roles, and they are often encouraged to learn from their more experienced instruction colleagues about topics such as pedagogy or assessment. For instruction librarians, the ACRL Framework for Information Literacy encourages librarians to look at the research process more holistically, an aspect that is often present in primary source instruction.  By interviewing department heads of both special collections and instruction teams at public and private academic libraries, the researchers gained insight into potential barriers that exist between the units, and aspirational goals for what a successful partnership would embody. This session will explore the opportunities, tensions, and values placed on instruction by both units.  Participants will be prompted to reflect on their own collaborative experiences within their libraries.
date: 2021-07-01 08:00:00
audience: IIG, SCAIG
keywords: Library instruction, information literacy, primary source literacy, collaboration, barriers, aspirations, communication
presenter-names: Kayla Harris,Heidi Gauder,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Exploring Social Justice through Metaliteracy and Algorithms 
description: "With the increased proliferation of artificial intelligence, algorithms play an unavoidable role in our lives. While many trust the algorithms without question, there is growing scholarship about the biases and imperfections within them.  Because of the relationship between algorithms and information seeking, it has implications on information literacy education. What do we know about the algorithms and their influences on results and user behavior? How does metaliteracy fit into the equation? \n\nThis poster looks to address these questions of algorithmic influence and metaliteracy through two means. First, it investigates the literature exploring the sources of algorithmic injustice and provides concrete examples. Then, it applies metaliteracy to algorithmic influences, which results in an action plan. Because metaliteracy is a foundational concept for the information literacy framework, it brings connection between librarian instruction goals and a trending societal concern.  "
date: 2021-07-01 08:00:00
audience: IIG, Library administration/supervision, diversity, consortia, emerging technologies, reference
keywords: algorithms, social justice,  metaliteracy 
presenter-names: Catie Carlson,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Beyond the Land Acknowledgement: Indigenous Language Revitalization, Student Activism, and Library Research
description: Throughout their history, libraries have participated in white supremacist power structures that privilege white knowledge over that of indigenous cultures. Indigenous language revitalization, which seeks to restore the languages of indigenous peoples, is one way libraries can begin to dismantle white supremacist culture. Through the lens of critical theory, this poster will examine a case study involving a collaboration between the library, the natural history museum, and a class of first-year students. This class focused on restoring a botanical walking tour and, as a direct result of student activism, incorporated indigenous language into newly created exhibit labels using an online dictionary of tribal words created by an indigenous tribe and the university that now sits upon their lands. During their research, students realized that many of the trees currently on campus are native to other parts of North America that were home to other indigenous peoples and thus are not represented in this dictionary, presenting significant challenges for the project and revealing the lasting effects of colonialism. This poster will discuss how such language suppression was used as a tool of white supremacist culture and how libraries can participate in social justice work through student research and activism.
date: 2021-07-01 08:00:00
audience: PROMIG, SCAIG, STEMIG, Library administration/supervision, diversity, consortia, emerging technologies, reference
keywords: Indigenous Language Revitalization, Critical Theory, Colonialism, White Supremacist Culture, Exhibition Labels, Language Suppression, Library/Museum Partnerships, Student Activism, Social Justice
presenter-names: Stefanie Hilles,Ginny Boehme,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Thinking through Diversity in the University’s Music Library to Connect to the Academic Community and Beyond
description: I am using my positioning as a woman of color, as well as my cultural studies background, to better inform what I exhibit and what I post on Twitter on behalf of the music library where I work. To contest assumptions that relegate Native Americans to the past, my student, who is Native American, and I focused on two current Native American musicians who also do philanthropy. While doing a focused exhibits on marginalized groups or individuals is important, having them in conversation with others in their time period or genre can also provide much needed context. When I did our exhibit on dance crazes, I used a multitude of artists and different time periods. Then I highlighted women and artists of color in my tweets about the exhibit. People may think they do not know any song from a Jamaican Reggae chanteuse, but they know the Electric Slide. Our collection’s constraints, centering maleness and whiteness, does provide challenges. For instance, our collection’s well-known Asian artists mainly perform classical music or are not American so I had to research more rigorously for our exhibit on Asian American artists. My goal is to center marginalized people and illustrate their agency.
date: 2021-07-01 08:00:00
audience: AIG, PROMIG, SCAIG
keywords: marginalized groups, racism, Orientalism, collaborations, accessibility
presenter-names: Trinidad Linares,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Illegal Searching: Solutions to Problematic Subject Headings
description: "A common example of systematic privilege in library settings involves the continued use of outdated subject terminology in cataloging. Perhaps the most famous example of this, the Library of Congress Subject Heading “Illegal alien” refers to undocumented citizens in a demeaning and derogatory manner, dehumanizing these individuals in a way that has been contested by library associations across the country for years.\n\nWhile it might be simple to find and report offensive subjects and to extrapolate on their faults, enacting changes in integrated library systems and in catalogs often proves a slow and forgotten task, especially considering the length of time that many of these terms have been used for cataloging purposes. At one university library, the discussion of this topic has become paramount due to the continued interest between it and its cultural advocacy challenges.\n\nThis presentation discusses possible solutions to allow users to search catalogued items despite - or perhaps because of - problematic or offensive subject headings, from including regulated note fields in MARC records to discussing systematic changes to official terminology."
date: 2021-07-01 08:00:00
audience: TEDSIG,Library administration/supervision, diversity, consortia, emerging technologies, reference
keywords: Diversity, subject headings, LCSH, metadata, RDA, inclusion
presenter-names: Justina Kaiser,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Hybrid by Design: New Student Orientations and the Value of Asynchronicity
description: "Covid changed our lives, our campuses, and how we connect with students. In-person activities were replaced with virtual ones and library events had to be reimagined digitally. With vaccinations on the rise and universities hopeful for a more in-person fall, what did we learn from online events that we can apply in the future?\nThis poster will present our experience transitioning the library’s annual graduate student orientation from in-person to virtual and hybrid events. First, we will discuss how we moved orientation online in 2020 and the challenges we faced, including teaching staff how to use new video conferencing software to record accessible videos, collaborating with campus partners, and how to best present the event with existing learning management platforms and the library website. Next, we will detail how we incorporated these lessons into a hybrid orientation in 2021 that will continue to use asynchronous recordings on an event landing page. Finally, we will discuss the benefits of asynchronous orientation elements and compare attendance outcomes from 2019 (in-person), 2020 (virtual), and 2021 (hybrid) to determine which format produces the most student engagement, answering the question, “Should we continue asynchronous opportunities at new student orientations when in-person learning fully resumes?” "
date: 2021-07-01 08:00:00
audience: PROMIG
keywords: library orientations, graduate students, virtual events, hybrid events
presenter-names: Abigail Morgan,Stefanie Hilles,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Charting our course, Measuring our progress: Creating a Data Dashboard for Library Services
description: Libraries and their administrations seek to demonstrate a return on investment made by our colleges or universities in our services and resources. Developing a dashboard which tracks and displays usage can be a helpful tool to assist with this task. This poster session will provide an overview of one attempt to develop a dashboard for circulation, document delivery, and information transactions. These experiences included: synthesizing data from multiple software / online services, each with its own idiosyncratic, and often inconsistent, interface; selecting tools for creating charts and graphs; selecting data to be charted over time and/or displayed in a summary view. The session is intended to benefit both those who work directly with data or from administrators who use data in decision-making. Take-homes may include: discussion of one model currently in use to develop models for an approach, we well as possible improvements; and an ability to evaluate pro’s and con’s of widely available services for manipulating and displaying data in order to select the most effective approaches.
date: 2021-07-01 08:00:00
audience: AIG, SSIG, Library administration/supervision, diversity, consortia, emerging technologies, reference
keywords: Dashboards, Statistics, Information Transactions, CIrculation, Document Delivery
presenter-names: 'Rob O''Brien Withers,'
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Transforming Research and Course Guides for Social Work
description: The COVID-19 pandemic has led library professionals to work in close collaboration and be more creative and innovative in digitally delivering their resources and services. This session discusses the pilot project between two library departments to transform research and course guides for Social Work. Although librarians spend considerable efforts creating these guides, students often have problems navigating, locating, and using them. The presenters will share the factors attributing to the low and limited use of these guides, such as the overwhelming number of hidden guides, overloaded resources without a learning context, partial duplication and similarity among siloed guides. The presenters will also describe transforming the Social Work research and course guide into a one-stop-shop with a creative approach. By the end of this session, attendees will learn how to pull various resources, embed interactive service points, integrate the guides via multiple venues, and facilitate increased faculty collaboration to better meet students at the point of their needs.
date: 2021-07-01 08:00:00
audience: C2YCLIG, IIG
keywords: course guides, research guides, library guides, collaboration, COVID-19 pandemic, social work
presenter-names: Dipti Mehta,Xiaocan (Lucy) Wang,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title:  Building Awareness: Connecting Faculty to Library Streaming Video Resources 
description: "As part of a review of our streaming video collections, I conducted a survey to learn more about our faculty’s use of streaming video, and their perceptions and awareness of the library’s role in providing access to this content. The findings included that 41% of respondents were unaware that the library provided access to streaming video collections. The majority of respondents indicated that they did not know the library had funds available to support streaming. Conversely, 76% of respondents reported that they were using streaming video for instruction, with 65% using commercial services. \nThe findings suggested a demand for streaming video, but less awareness of the resources we offered. To better position the library as a resource for quality streaming video, we conducted a streaming video workshop for our faculty. We aimed to provide a high level overview of many common issues surrounding streaming such as licensing, access, copyright, open educational resources, and effective pedagogy. In this poster, I will share our library’s experiences and offer strategies for engaging with faculty about streaming video resources, including how to establish your library as a source for stable, quality access, and how to market the expertise of your librarians on this topic. \n"
date: 2021-07-01 08:00:00
audience: AIG, CMIG, TEDSIG
keywords: streaming video, faculty outreach
presenter-names: Katy Tucker,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Patents and Privilege:  Using patents to teach that information has value
description: "Patents are not only useful for STEM fields, but they can also be intriguing primary source documents for researchers.  They describe innovation, reference other inventions, and provide a timeline of the history of technology in the U.S.  Our founding fathers valued innovation so much that they included protections in the U.S. Constitution.  For these reasons and more, the librarians at this academic library developed two workshops that introduced students to the value of patents, how to research patents, and their place in our city’s own history of innovation.\n\nHowever, although patents protect innovation, in doing so, they have also historically privileged those inventors who are in a position to create and to afford the application fees.  As part of the workshop, students were asked to map the addresses of the inventors, the results of which played along race and class lines across the city.  These results facilitated a discussion about the lack of representation in the patents and why.  Using the ACRL Information Literacy frame, Information has value, this poster will show attendees how patent history reads as a history of exclusion but also an opportunity to discuss the complexities of patents, research, and innovation.\n"
date: 2021-07-01 08:00:00
audience: IIG, PROMIG
keywords: patents, privilege, digital humanities, primary sources
presenter-names: Heidi Gauder,Bridget Retzloff,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Supporting academic library worker mental health through flexible and remote work
description: This poster provides an overview of relevant research on flexible and remote work schedules and how they impact job satisfaction and work/life balance. The poster focuses specifically on the implications of flexible and remote work for academic library workers, particularly those living with mental health challenges/illnesses. It offers suggestions about how libraries can support library workers by increasing opportunities to work remotely and flexibly. 
date: 2021-07-01 08:00:00
audience: Library administration/supervision, diversity, consortia, emerging technologies, reference
keywords: Mental health and library workers, flexible work, remote work, work life balance
presenter-names: Jillian Ewalt,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Crossing the Bridge from In-Person to Online: Write on Site to Writing Together
description: Purpose: For librarians with scholarly activity requirements, regular work on these activities fosters success. A weekly writing group can provide support and accountability for these librarians. This poster describes a transition from an in-person to an online writing group. Brief Description: In 2019, the [redacted] library at [institution] began holding weekly 60-minute “Write on Site” sessions to provide structure and informal accountability. Participants could attend sessions as needed and could do whatever was needed to move a project forward. In March 2020, Write on Site was cancelled due to the pandemic, and in January 2021, it was reborn as 90-minute online Writing Together using Microsoft Teams. Results: For the in-person writing group, we had three unique participants. All reported benefits of getting together each week to work on their writing. For the online writing group, participation has been higher (eight unique participants) because people could connect from any location. Online meetings facilitated connecting with colleagues at other institutions. The challenges of an online writing group included connection issues and "Zoom fatigue." Conclusion: Because of the benefits, the online format will be maintained after it is safe to meet in person.
date: 2021-07-01 08:00:00
audience: Library administration/supervision, diversity, consortia, emerging technologies, reference
keywords: writing groups, scholarly activity, online meetings
presenter-names: Jolene Miller,Margaret Hoogland,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: A Celebration of Us!:  A Telling of a Libraries’ Value During COVID-19 Through Stories and Data
description: "The university libraries accomplished many new and innovative services and projects through the pandemic with an extremely limited staff and decreasing budgets.  Think about what the libraries could do if more investments were made to advance our excellent services, resources, and personnel. \n\nThis past year and a half have been like no other, and the large, urban library went above and beyond to serve the its community during this time.  A library COVID Data Group was charged to gather data regarding services that demonstrated its value to the university’s mission.\n\nThe timeframe for the data was March 13, 2020 to April 2021.  The Group emailed key staff to gather quantitative data and sent out a survey to all staff to collect qualitative data.  The Group analyzed that data and wrote a report.  The report included data and stories that celebrated all of the new services and hard work that the staff performed over the time of the pandemic. in the areas of student/faculty success, teaching/learning, innovative solutions, online collections, and scholarly content. The findings proved to be an excellent marketing tool and benefitted the library by enabling the Deans and librarians to engage high-level audiences in meaningful conversations. \n"
date: 2021-07-01 08:00:00
audience: PROMIG, SUSIG, Library administration/supervision, diversity, consortia, emerging technologies, reference
keywords: COVID-19 ; Value of Libraries; Storytelling Methods; Data Gathering and Analyzing Techniques
presenter-names: Deborah Tenofsky,Ted Baldwin,Catie Carlson,Katie Foran-Mulcahy
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Life as a Sandwich Artist! The Role of Librarian as Middle Manager in Leading Change
description: "Middle managers in libraries are leaders of organizational change! But this was not always the case. Decades ago, middle managers were task masters, serving in the most vulnerable, sacrificial positions of an organization. Middle managers are still sandwiched between administration and staff, but the emerging trend shows them as leaders of organizational change. They now serve as the platform and catalyst for enabling our libraries to survive, succeed, and thrive in the continuous evolution of change.\nAlthough still embedded between library administration and library staff, middle managers in libraries are no longer charged simply with managing people and their work. Instead, they acquire and apply a specific set of leadership skills – networking, influence, communication, and alignment – to manage complex, interdependent events with accountability and authority. Middle managers in our libraries are sandwich artists! This presentation will highlight the unique and multi-faceted skills of middle managers and share why these skills are fundamental for leading change in our libraries.\n"
date: 2021-07-01 08:00:00
audience: AIG, Library administration/supervision, diversity, consortia, emerging technologies, reference
keywords: Library Managers, Management, Middle Management, Middle Managers, Leadership Skills, Organizational Change, Change Management
presenter-names: Christine Tobias,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Connecting across Campus by Amplifying Anti-Racism Resources
description: In summer 2020, [redacted institution name] Libraries was asked to partner in collaboration with [redacted name institution] Office of Diversity and Inclusion (ODI) to provide expanded access to anti-racism book content in support of continuing education, reading groups, and professional development across the university. The Libraries’ African American and African Studies Librarian worked with ODI’s Director for Strategic Diversity Planning, Training, and Assessment to identify anti-racism book titles of interest that were recommended for purchase or expanded access to the Collections Strategist. Topical research guides were created to supplement the reading lists, and new collections practices were developed and implemented to support campus-wide anti-racism educational opportunities. This poster will detail not only the process of collaboration with another unit within the university, but outline challenges faced with budget management, publishing and vendor practices with anti-racism resources, and COVID-19 implications. We will address critical approaches to anti-racism within library collaborations to university campus communities including outreach, teaching, and collections as part of the evolving Diversity, Equity, Inclusion, and Accessibility (DEIA) movement.
date: 2021-07-01 08:00:00
audience: CMIG, SUSIG, Library administration/supervision, diversity, consortia, emerging technologies, reference
keywords: Anti-racism   Diversity, Equity, Inclusion, and Accessibility (DEIA)  Acquisitions 
presenter-names: Leta Hendricks,Gene R. Springs ,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
---
layout: poster
title: Enhancing student engagement using story-based videos
description: In response to the evolving changes in the teaching and learning environment, we re-designed several learning videos to encourage a more engaging learning experience by incorporating new instructional strategies, specifically story-based learning. In contrast to conventional screen-recorded video tutorials with narratives, story-based videos were developed with cartoon characters, settings, and contexts similar to student life. Both types of videos were used for library instruction. Informal assessments have shown that both types of tutorials were effective in terms of student learning outcomes. We conducted a user study with 50 first-year students with two kinds of videos to address student engagement. The poster will share the detailed results and comments from students. The main takeaways are that while the immediate learning outcomes were similar, student engagement and confidence level were higher for the story-based video. And, when students were asked about their preferences of presentation type if given an option, the majority of them choose story-based presentation. This study tests the approach in an academic library setting with undergraduate students and potentially can lead to improvements in information literacy programs and inform library practices. We believe the story-based approach has great potential to enhance student learning in an online setting as well. 
date: 2021-07-01 08:00:00
audience: AIG, DLIG, IIG
keywords: student engagement; user experience; video tutorials
presenter-names: Feng-Ru Sheu,
speaker-data: 
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
```

**Comment by kenirwin** [comment]
@tuckerk3 - I'd like to ask you to do the next stage of this project. it will involve creating a bunch of small files, and adding presenter ids similar to what you did in the sessions.yml file.

unlike sessions, posters are created by make one file per poster in the _posts directory. each file has a format like this: 

```
---
layout: poster
title: Enhancing student engagement using story-based videos
description: In response to the evolving changes in the teaching and learning environment, we re-designed several learning videos to encourage a more engaging learning experience by incorporating new instructional strategies, specifically story-based learning. In contrast to conventional screen-recorded video tutorials with narratives, story-based videos were developed with cartoon characters, settings, and contexts similar to student life. Both types of videos were used for library instruction. Informal assessments have shown that both types of tutorials were effective in terms of student learning outcomes. We conducted a user study with 50 first-year students with two kinds of videos to address student engagement. The poster will share the detailed results and comments from students. The main takeaways are that while the immediate learning outcomes were similar, student engagement and confidence level were higher for the story-based video. And, when students were asked about their preferences of presentation type if given an option, the majority of them choose story-based presentation. This study tests the approach in an academic library setting with undergraduate students and potentially can lead to improvements in information literacy programs and inform library practices. We believe the story-based approach has great potential to enhance student learning in an online setting as well. 
date: 2021-07-01 08:00:00
audience: AIG, DLIG, IIG
keywords: student engagement; user experience; video tutorials
presenter-names: Feng-Ru Sheu,
speaker-data: [21]
session-contents: 
supplemental-docs: 
isStaticPost: false
published: true
---
```

To proceed with this project

* start by using GitHub Desktop to checkout the `posters` branch -- it's already got a bunch of changes I've made but that shouldn't be merged until this next part is complete. 
* On that branch, take a look at the sample file in the `_posts` directory; you'll make a bunch of files like it. 
* one file at a time, create a file with a name like: 2021-07-01-meaningful-slug
  * use the 2021 date listed here for this year's files; the july 1 date isn't important but let's just keep them all the same.
  * replace "meaningful-slug" with some lower case, hyphenated stuff to give a short title for the post based on the poster title. (Last year "Connecting the Past to the Present: Alumni Engagement with the Library During Homecoming" became "alumni-engagement"; "Effects of high school location on first-year students’ research confidence and college readiness" -> "hs-location" -- it's a handy label, not an extensive summation
* in one of the [comments above](https://github.com/alaoweb/2021.alaoweb.org/issues/78#issuecomment-885837302), I put in most of the data for each file -- be sure to include the three dashed lines at the beginning and end of each file; they are weirdly important, so:
```
---
some: data
more: data
---
```
* finally, add an array for speaker data like before:
```
speaker-data: [21]
```
or 
```
speaker-data: [21, 17]
```

As you build the files you you see the posters page repopulate with cards for each poster and names attached to each.

Does that sound workable?

**Comment by tuckerk3** [comment]
@kenirwin Sounds good. Thanks!

**Comment by kenirwin** [comment]
closed by #93
